src/cunumeric/cuda_help.h

```diff
@@ -108,14 +108,41 @@ class cufftContext {
   std::vector<cufftXtCallbackType> callback_types_{};
 };
 
+struct cufftPlanParms {
```

Comment:
a nitpick: "parms" seems a bit unusual abbreviation for "parameters". ("params" would be a typical name)